docs/fileio_usage.rst

```diff
@@ -3,60 +3,104 @@ Usage
 =====
 
 The easiest way to use :py:mod:`orsopy.fileio` (the module of :py:mod:`orsopy` that includes file reading and writing) to produce **metadata-rich .ort reduced reflectometry files** involves integrating this into your data reduction workflow.
-Early in the workflow, the :py:mod:`orsopy.fileio` should be imported and an empty :code:`orsopy.fileio.orso.Orso` header object (here we also import :py:mod:`numpy` which will be used later). 
+Early in the workflow, the :py:mod:`orsopy.fileio` should be imported and an empty :code:`orsopy.fileio.orso.Orso` header object (here we also import :py:mod:`numpy` which will be used later).
 
-.. code-block:: python 
+.. code-block:: python
 
-    import numpy as np
     from orsopy import fileio
+    from orsopy.fileio import (Reduction, Person, Orso, Experiment,
+                               Sample, DataSource, Measurement, InstrumentSettings)
+    import numpy as np
+    import datetime
 
     header = fileio.orso.Orso.empty()
 
-Having created the empty header object we can start to populate the appropriate components of it. 
-It is generally a good idea to populate the components as particular steps occur in the reduction process. 
-For example, if we want to identify the probing radiation as neutrons, we include this as follows. 
+Having created the empty header object we can start to populate the appropriate components of it.
+It is generally a good idea to populate the components as particular steps occur in the reduction process.
+In this example we fill out the experiment details and then wrap them in the appropriate class.
 
-.. code-block:: python 
+.. code-block:: python
 
-    header.data_source.experiment.probe = 'neutron'
+    title = 'my title'
+    instrument = 'PLATYPUS'
+    start_date = datetime.date.today()  # This needs to be a datetime object
+    probe = 'neutron'
+    facility = 'ESS'
+    proposalID = 999999
+    doi = f'10.5286/ISIS.E.RB{99999}'
```

Comment:
Have you deliberately mixed ISIS, ESS and ANSTO?

Reply:
yes